src/pages/product.html

```diff
@@ -0,0 +1,227 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="../styles/output.css">
+  <title>Product</title>
+</head>
+
+<body>
+  <header class="header">
+    <img class="logo" src="../assets/icons/Finsweet black logo.svg" alt="FinSweet black logo">
+    <nav class="header__nav-links">
+      <a href="../index.html" class="link">Home</a>
+      <a href="#" class="link">Product</a>
+      <a href="#" class="link">Pricing</a>
+      <a href="#" class="link">About us</a>
+      <a href="./blog.html" class="link">Blog</a>
+      <a href="./contact.html" class="link">Contact</a>
+      <button type="button" class="button button--primary">Free trial</button>
+    </nav>
+  </header>
+
+  <section class="pt-24 pb-32 bg-[url('../assets/backgrounds/Blur_Gradient_product_hero.png')] bg-cover">
+    <section class="halved-section">
+      <div class="section-text">
+        <h2 class="section-text__heading">Grow your Sales and Services</h2>
+        <p class="description desciption--why-us-section">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed
```

Comment:
`desciption--why-us-section` is included without it's default Element (desciption).